.github/workflows/build.yml

```diff
@@ -4,28 +4,18 @@ on:
     branches-ignore:
       - main
       - gh-pages
-
 jobs:
   build-and-test:
-    timeout-minutes: 60
     runs-on: ubuntu-latest
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v4.2.2
       - uses: actions/setup-node@v4
         with:
           node-version: 22
-      - name: Install dependencies
-        run: npm ci
-      - name: Build
-        run: npm run build
-      - name: Install Playwright Browsers
-        run: npx playwright install --with-deps
-      - name: Run Playwright tests
-        run: npx playwright test
-      - uses: actions/upload-artifact@v4
-        if: ${{ !cancelled() }}
+      - name: Build and Run End2End tests with Cypress
+        uses: cypress-io/github-action@v4
         with:
-          name: playwright-report
-          path: playwright-report/
-          retention-days: 30
+          build: npm run build
+          start: npm run dev
+          wait-on: "http://localhost:8080"
```

Comment:
This looks off, those seem to be unintended reverts.

Reply:
right. Some rebase error. I will revert it